docs/configuration.rst

```diff
@@ -24,10 +26,21 @@ should export your Stormpath information by running this in the shell:
     export STORMPATH_CLIENT_APIKEY_SECRET=YOUR-SECRET-HERE
     export STORMPATH_APPLICATION_HREF=YOUR-APP-HREF
 
+You might also want to check out
+`autoenv <https://github.com/kennethreitz/autoenv>`_, a project that makes
+working with environment variables simpler for Linux / Mac / BSD users.
+
 .. note::
-    If you're on Windows, that will look like::
+    If you're on Windows, the command you need to use to set environment
+    variables is::
 
         set STORMPATH_CLIENT_APIKEY_ID=YOUR-ID-HERE
+        set STORMPATH_CLIENT_APIKEY_SECRET=YOUR-SECRET-HERE
+        set STORMPATH_APPLICATION_HREF=YOUR-APP-HREF
+
+The examples above show you the 3 mandatory settings you need to configure to
+make express-stormpath work.  These settings can be configured via environment
+variables, or in a number of other ways.
```

Comment:
Good clarification, I believe this paragraph should be moved below the code block that shows the export examples.